### PR TITLE
Build not only x64

### LINKED ISSUE #46: Fix image on arm64
since I've switched to an apple silicon notebook, I have the problem that this image doesn't work anymore. Docker does start it a amd64 compatibility environment (Qemu) that tries to do it's thing, but sadly fails.

```
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
```

Interestingly all the other Icinga images seem to work fine under emulation.

It may be easy to fix this (and I'm willing to help debug this) but it may be a better solution to build a multi-platform image that supports arm64 natively. This should also allow for much better performance.

This seems to be some documentation on [how to build multi arch images](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/) ([buildx documentation](https://docs.docker.com/buildx/working-with-buildx/)).  `docker buildx --platform ...` seems to be the way to go.

### PR DESCRIPTION
**Pull request by Al2Klimov**

### CODE REVIEW

**Comment by Al2Klimov** [comment]
## TODO

* Icinga/docker-icinga2#73
* [x] create all tags
* [x] adjust GHA usage

**Comment by Al2Klimov** [comment]
Test pipeline: https://github.com/Icinga/icingadb/actions/runs/3938296860/jobs/6744592491

**Comment by Al2Klimov** [comment]
## TODO

* [ ] go mod dl only 1x

**Comment by Al2Klimov** [comment]
> ## TODO
> * [ ]  go mod dl only 1x

Tbh. I already did literally the same here:

https://github.com/Icinga/docker-icinga2/blob/faecd36aa647f294a6a35114ee663889ba520ae7/Dockerfile#L14-L22

🤷‍♂️

**Comment by Al2Klimov** [comment]
A clean Icinga DB v1.2.0 thing for me.

**Comment by Al2Klimov** [comment]
@julianbrost I have to correct myself:

A clean Icinga DB ~v1.2.0~ v1.1.1 thing for me.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -15,10 +15,13 @@ jobs:
       - name: Checkout HEAD
         uses: actions/checkout@v1
 
+      - name: QEMU
+        run: sudo apt-get install -y qemu-user-static binfmt-support
```
Comment (julianbrost) [inline .github/workflows/build.yml:22]: Misses this change from https://github.com/Icinga/docker-icinga2/pull/99

File: action.yml [hunk 2]
```diff
@@ -9,6 +9,9 @@ runs:
   steps:
     - uses: actions/checkout@v3
 
+    - shell: bash
+      run: sudo apt-get install -y qemu-user-static binfmt-support
```
Comment (julianbrost) [inline action.yml:16]: Misses this change from https://github.com/Icinga/docker-icinga2/pull/99

File: platforms.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x
```
Comment (julianbrost) [inline platforms.txt]: It doesn't really make sense to build for more platforms than [we build Icinga 2 containers for](https://github.com/Icinga/docker-icinga2/blob/master/platforms.txt).

**Review by julianbrost — CHANGES_REQUESTED** [review]

**Comment by julianbrost** [comment]
> A clean Icinga DB ~v1.2.0~ v1.1.1 thing for me.

A clean "when there is time for it" thing. Like if it was very important, we could also upload new images for an existing release with additional architectures.

**Review by julianbrost — APPROVED** [review]

**Comment by dwt** [comment]
🎉

**Pull request merged**